test-data/local/snake_oil_structure/ert/model/user_config.ert

```diff
@@ -0,0 +1,50 @@
+--===============================================================
+--
+-- ERT CONFIGURATION FILE
+--
+--===============================================================
+
+---------------------------------
+-- SECTION: Internal variables --
+---------------------------------
+DEFINE 	<USER> 		TEST_USER
+DEFINE 	<SCRATCH> 	scratch/ert
+DEFINE 	<CASE_DIR>  the_extensive_case	
```

Comment:
Oh, will do! That is what you get for using an existing configuration file as a starting base..